build.rs

```diff
@@ -0,0 +1,39 @@
+fn main() {
+    let out_dir = std::env::var("OUT_DIR").unwrap();
+    let target = std::env::var("TARGET").unwrap();
+
+    let machine = if target.contains("x86_64") {
+        "X64"
+    } else if target.contains("ARM64") {
+        "ARM64"
+    } else if target.contains("ARM") {
+        "ARM"
+    } else if target.contains("i686") {
+        "X86"
+    } else {
+        panic!("Unsupported target");
+    };
+
+    let lib = cc::windows_registry::find_tool(&target, "lib.exe").expect("lib.exe not found");
```

Comment:
I'm assuming `lib.exe` would not be there if VS build tools aren't installed? Perhaps we could make a nice error message pointing people where to install them?